src/main/java/mate/academy/spring/boot/model/ShoppingCart.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.spring.boot.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.MapsId;
+import jakarta.persistence.FetchType;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Getter
+@Setter
+@ToString
```

Comment:
Be careful with using lombok's toString on JPA entities, you have lazily loaded depended entities, which can cause lazy init exception, use @ToString(exclude = "cartItemSet")